packages/core/src/prompts/multi-select.ts

```diff
@@ -4,6 +4,7 @@ import color from 'picocolors';
 interface MultiSelectOptions<T extends { value: any }> extends PromptOptions<MultiSelectPrompt<T>> {
     options: T[];
     initialValue?: T['value'];
+    isAllowEmpty?: boolean
```

Comment:
Would love if this was named `required` but defaulted to `true`, so this behavior would be covered by `required: false`

```suggestion
    required?: boolean
```